### PR TITLE
[DOCFIX] add one command in cn docs

### PR DESCRIPTION
**Pull request by xuerenlv**
NanJing University Hackathon

https://alluxio.atlassian.net/browse/ALLUXIO-1978

### CODE REVIEW

**Comment by ifcharming** [comment]
Thanks @xuerenlv for the translation! The PR looks great.

Per Alluxio convention, can you please add `[DOCFIX]` in the PR title as the prefix?
If there's any JIRA ticket for this, you can also add the ticket link in the PR description.

**Comment by ifcharming** [comment]
ok to test

File: docs/cn/Configuring-Alluxio-with-GlusterFS.md [hunk 1]
```diff
@@ -12,7 +12,11 @@ priority: 2
 
 首先，本地要有Alluxio二进制包。你可以自己[编译Alluxio](Building-Alluxio-Master-Branch.html)，或者[下载二进制包](Running-Alluxio-Locally.html)
 
-然后，如果你还没有生成配置文件，可以由template文件创建配置文件：
+然后，如果你还没有去配置，你可以通过 `bootstrap-conf` 命令进行配置。例如：如果你在本地运行 Alluxio ，你要把 `ALLUXIO_MASTER_HOSTNAME` 设置为 `localhost`。
+
+{% include Configuring-Alluxio-with-GlusterFS/bootstrap-conf.md %}
+
+或者，如果你还没有生成配置文件，可以由template文件创建配置文件：
```
Comment (ifcharming) [inline docs/cn/Configuring-Alluxio-with-GlusterFS.md]: how about 
"或者，你也可以利用template文件创建配置文件并且手动设置配置内容。" ?

**Comment by ifcharming** [comment]
thanks @xuerenlv , I left one minor comment, otherwise LGTM

**Comment by ifcharming** [comment]
LGTM, thanks for the improvement!

**Comment by ifcharming** [comment]
Can you please add the JIRA ticket URL (https://alluxio.atlassian.net/browse/ALLUXIO-1982) in the PR description?

**Comment by ifcharming** [comment]
Another PR (https://github.com/Alluxio/alluxio/pull/3465) was issued for the same ticket after this one.
So please make sure to grab the JIRA ticket first (assign it to yourself) and then start working on it. Otherwise multiple PRs could work on the same ticket...
Also it would help avoid conflicting by adding the JIRA ticket URL in the PR description. This way, the PR is automatically linked in the corresponding JIRA ticket.

**Comment by xuerenlv** [comment]
very sorry! this is the first time . should i close it ?

**Comment by ifcharming** [comment]
Sorry, never mind my previous message, the other PR was for a different ticket (for swift), this one is for GlusterFS.

Please just add the corresponding JIRA ticket in the PR description and title :)
thanks!

**Comment by xuerenlv** [comment]
ALLUXIO-1978
Add the bootstrap-conf command snippet in Configuring-Alluxio-with-GlusterFS.md of the Chinese version according to the English version for Alluxio 1.1

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**